Gemfile

```diff
@@ -7,7 +7,7 @@ end
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 5.1.5'
+gem 'rails', '~> 5.2.0.rc2'
```

Comment:
We discussed this a while ago - its so we can use active storage.  Maybe you were away at the time.  We were taking a risk - hoping that the new version would be released before go live - and as luck would have it - it was released this morning, so there will be another PR coming along shortly to upgrade to new version.

Reply:
Indeed, we had the discussion to go with the RC version back then and Rails 5.2.0 FINAL was release just few days ago. 

Since this PR is still open, it makes sense to me to upgrade it now. Should be quick.

Reply:
ok , cool

Reply:
Now upgraded to 5.2.0 :-)